Sources/Core/JavaIR.swift

```diff
@@ -49,18 +49,34 @@ public struct JavaIR {
     }
 
     public struct Property {
+        let annotations: Set<String>
         let modifiers: JavaModifier
         let type: String
         let name: String
+        let initialValue: String
+
         func render() -> String {
-            return "\(modifiers.render()) \(type) \(name);"
+            var prop = "\(annotations.map { "@\($0)" }.joined(separator: " ")) \(modifiers.render()) \(type) \(name)"
+            if !initialValue.isEmpty {
+                prop.append(" = " + initialValue)
+            }
+            prop.append(";")
+            return prop
         }
     }
 
     static func method(annotations: Set<String> = [], _ modifiers: JavaModifier, _ signature: String, body: () -> [String]) -> JavaIR.Method {
         return JavaIR.Method(annotations: annotations, modifiers: modifiers, body: body(), signature: signature)
     }
 
+    static func ifBlock(condition: String, body: [String]) -> String {
```

Comment:
so if you check out the DSL for ObjC, we usually specify the `body` as a closure that returns an array of strings `() -> [String]`. This allows you to write things like 
```
JavaIR.ifBlock("someCondition") {[
  "First line of body",
  "Second line of body",
 ....
]}
```

Reply:
Done.